my-app/src/components/calendar/calendar.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Does this file pass eslint checks? I feel like the indentation may be off. If you already ran eslint ignore this comment.